po/es.po

```diff
@@ -289,7 +289,7 @@ msgstr "Manejo de la Memoria en Rust"
 
 #: src/SUMMARY.md:88 src/ownership.md:1
 msgid "Ownership"
-msgstr "Propiedad (_Ownership_)"
+msgstr "Ownership"
```

Comment:
We had a very in depth discussion at https://github.com/google/comprehensive-rust/discussions/1184 , and not sure that we had full consensus, but I think we pretty much came up with `translated_msg (_english_msg_)`

If this is a formatting concern due to the underscore, should we just use `translated_msg (english_msg)`  instead?

Reply:
I don't oppose the idea of translating certain terms, as it's a topic of discussion in some Discord and Telegram channels. Many individuals support the notion of retaining the original English term, but it doesn't hurt to include a translated alias.

I concur with this perspective because it provides an additional means for users to grasp the terminology. For those who are not well-versed in English, comprehending the terms can be challenging.

In this case:
> [¿Qué es el Ownership?](https://rustlanges.github.io/rust-book-es/ch04-01-what-is-ownership.html)

We employ the term "Ownership" but provide a note stating that "Propiedad" is an acceptable alternative for conveying the same concept.

I intend to revert this change in order to maintain both options.

Reply:
The underscores are a problem here, as mentioned in #1328. So they should go for sure.

Reply:
Actually, you can keep the formatting! I'm working on making it look correct in https://github.com/google/mdbook-i18n-helpers/pull/100. It should be merged in a few days and then formatting becomes file in the summary file.